editor/store.js

```diff
@@ -27,6 +28,7 @@ function createReduxStore() {
 	const enhancers = [
 		applyMiddleware( multi, refx( effects ) ),
 		storePersist( 'preferences', GUTENBERG_PREFERENCES_KEY ),
+		applyMiddleware( mobileMiddleware ),
```

Comment:
I guess it's fine as it is, but I'm wondering why they do it differently in the official Redux docs: https://redux.js.org/docs/api/applyMiddleware.html#tips. In short, something like this:

```js
let middleware = [a, b];

const store = createStore(
  reducer,
  preloadedState,
  applyMiddleware( ...middleware )
)
```

Reply:
Redux documentation states that middleware should be applied before other enhancers, because enhancers may not see middleware changes otherwise. But this a specific case, we need this middleware to be applied after our storePersist enhancer because it changes storePersist. So I think because we need the middleware after the non-middleware enhancer we will not be able to use the same way used in redux docs.

Reply:
I think I get it now. I wasn’t aware of the concept of enhancers. I found an example how the should be used and it matches what we have: https://redux.js.org/docs/api/compose.html

Thanks for spending your time explaing what’s happening here 🙇🏻